agent/commands/run.go

```diff
@@ -94,17 +106,18 @@ func Run() {
 		logrus.Infof("Window check connection time is %.2f hour(s)", cfg.WindowConnectedTime.Hours())
 		connectionUptimeService.SetWindowPeriod(cfg.WindowConnectedTime)
 
-		clientCtx, cancel := context.WithCancel(ctx)
+		clientCtx, cancelClientCtx := context.WithCancel(ctx)
 
 		_ = client.Run(clientCtx)
```

Comment:
Good point, fixing

Reply:
Actually, we have only 2 places where we return an error in client.go and in one of them we log it, for the other one I suppose that we don't log context error because it's some kind of expected error